snippets/csharp.json

```
@@ -198,7 +198,7 @@
     "Interface": {
         "prefix": "interface",
         "body": [
-            "interface I${1:Name}",
+            "interface I${1:$TM_FILENAME_BASE}",
```

Do we expect the filename would already included the I prefix?

```suggestion
            "interface ${1:$TM_FILENAME_BASE}",
```

---

Oh, I must remove by accident :D

---

In default, every c# project has `I` in filename for example https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/ICloneable.cs

---

Would this template generate `interface IICloneable` for that file?

---

My bad, I fixed this